src/material-experimental/mdc-list/testing/list-harness-filters.ts

```diff
@@ -17,7 +17,10 @@ export interface NavListHarnessFilters extends BaseHarnessFilters {}
 export interface SelectionListHarnessFilters extends BaseHarnessFilters {}
 
 export interface BaseListItemHarnessFilters extends BaseHarnessFilters {
-  text?: string | RegExp;
+  title?: string | RegExp;
```

Comment:
can we retain `text` as a deprecated equivalent to `fullText`. That way apps that already adopted the harnesses can migrate to the MDC version and have their test still work

Reply:
yeah, I can do that. I was considering it but didn't go with it given the API has changed a lot anyway. Let me know.

Reply:
Lets keep this as deprecated. We've been telling people "use the harnesses, it'll make migration easier", so I'd like us to not be liars 😅

Reply:
haha, fair enough. will add it

Reply:
done.